src/modules/file-explorer/components/file-explorer.jsx

```diff
@@ -13,6 +13,44 @@ class FileExplorer extends React.Component {
 		super(props);
 	}
 
+	/**
+	 * split files-list in files, that are in current directory, and the sub-directories
+	 * @param data is the files-list
+     */
+	splitFilesAndDirectories(data) {
```

Comment:
The current "folder" content without subdirectories can be fetched in S3 by using a `/` delimiter (see <https://forums.aws.amazon.com/thread.jspa?messageID=465624>)